RequestManagementSystem/Client/Request.py

```diff
@@ -370,7 +370,7 @@ def Status( self, value ):
     # If the status moved to Failed or Done, update the lastUpdate time
     if value in ( 'Done', 'Failed' ):
       if value != self._Status:
-        self.LastUpdate = datetime.datetime.utcnow().replace( microsecond = 0 )
```

Comment:
Now, regarding that one, it is more of a best practice... if the internal argument is protected by property, it is because we are "doing something on it". So it is probably best to let it go through the property as well.